src/main.cpp

```
@@ -2,12 +2,11 @@
 #include "main_window.h"
```

Cppcheck (2.20.0)

[information] Include file

(missingInclude)